services/ops/EmailerAgent/README.md

```diff
@@ -0,0 +1,105 @@
+Emailer
+=======
+
+The Emailer agent allows an instance of the VOLTTRON platform to send
+email.  When used in combination with the Alert agent alerts from
+unpublished configured devices will automatically be sent.  In addition,
+agents are able to send emails directly through the pubsub interface.
+
+Agents needing to send an email through the instance can do so by
+sending the following header and message to the "emailer" topic.  The
+Emailer agent monitors the "platform_send_email" topic.  The following 
+is the expected payload for the message body and the optional header.
+
+
+Optional Headers
+----------------
+
+Emails by default will be sent to the initial configured email
+addresses. The below headers will overwrite those properties for the
+current email being sent.
+
+``` {.sourceCode .python}
+{
+    "from-address": 'foo@bar.com',
+    "to-addresses": ['alpha.beta@foo.com', 'bob-and-joe@bar.com']
+}
+```
+
+
+Required Message Body
+---------------------
+
+``` {.sourceCode .python}
+{
+    "subject": "I am a happy camper",
+    "message": "This is a big long string message that I am sending"
+}
+```
+
+
+Example Sending of Email
+------------------------
+
+``` {.sourceCode .python}
+headers = {
+    "from-address": 'foo@bar.com',
+    "to-addresses": ['alpha.beta@foo.com', 'bob-and-joe@bar.com']
+}
+
+message = {
+    "subject": "I am a happy camper",
+    "message": "This is a big long string message that I am sending"
+}
+
+self.vip.pubsub.publish('pubsub', topic='platform/send_email',
+                        headers=headers, message=message)
+```
+
+
+Configuration Options
+---------------------
+
+The following JSON configuration file shows all the options currently
+supported by the Forward Historian agent.
+
+``` {.sourceCode .python}
+{
+    # The smtp-address (Simple Mail Transfer Protocol) to ship the email
+    # from the "from-address" to each of the recipients.
```

Comment:
comma after "from" or parens around (the "from-address" to each of the recipients).